frontend/src/routes/Video.tsx

```diff
@@ -581,6 +593,22 @@ const MetadataTable: React.FC<MetadataTableProps> = ({ event }) => {
         ]);
     }
 
+    if (event.syncedData?.updated) {
+        const fullOptions = { dateStyle: "long", timeStyle: "short" } as const;
+        const { created, updated } = getEventTimeInfo(event as SyncedEvent);
+
+        const updatedFull = updated.getTime() - created.getTime() > 5 * 60 * 1000
+            ? updated.toLocaleString(i18n.language, fullOptions)
+            : null;
+
+        if (updatedFull != null) {
+            pairs.push([
+                t("video.updated"),
+                updatedFull,
+            ]);
+        }
+    }
```

Comment:
So you have added this to the metadata table, but you haven't remove the date popup. On our document for the video page, we wrote down some specifics on what dates we want to show in what cases. So yes, the "updated" date we wanted to always show. Does it make sense to show it in the metadata table then? I don't particularly mind, but if we have this hover-tooltip on the main-date anyway (for things like end time), then I don't see why the "updated" time shouldn't also live there? Or did you plan to also put those other dates into the metadata table and just forgot to remove the tooltip?

Reply:
Whats the status on this? The updated date seems to have disappeared from the metadata table but it's also not in the date tooltip?

Reply:
It should be there, and is on most videos. Maybe the ones you looked at haven't been updated?

Reply:
Oh yeah, not sure how I missed that.